readme.txt

```diff
@@ -180,6 +180,16 @@ Check out our extensive [knowledgebase](https://evnt.is/18wm) for articles on us
 
 == Changelog ==
 
+* Tweak - Changed the word `Purchaser` to `Attendee` when email confirmation is sent for purchasing a ticket. [ETP-655]
```

Comment:
This changelog seems to be int he wrong place.

Reply:
Hi @redscar, we need to always include the changelog entries below the upcoming release number, which is normally already set with the release date as TBD.

For this particular case it should be below the `= [5.1.6] TBD =`